### PR TITLE
Part plugin docs

### PR DESCRIPTION
**Pull request by vpodzime**
The first commit is a minor fixup of the *features.rst* file, the second one adds the actual documentation to the *part* plugin.

### CODE REVIEW

**Comment by vojtechtrefny** [comment]
Looks good to me.

File: src/plugins/part.c [hunk 1]
```diff
@@ -33,7 +33,18 @@
  * @title: Part
  * @include: part.h
  *
- * A plugin for operations with partition tables.
+ * A plugin for operations with partition tables. Currently supported table
+ * (disk label) types are MBR and GPT. See the functions below to get an
+ * overview of which operations are supported. If there's anything missing,
+ * please don't hesitate to report it as this plugin (just like all the others)
+ * is a subject to future development and enhancements.
```
Comment (snbueno) [inline src/plugins/part.c]: "is subject to...."

**Comment by snbueno** [comment]
Looks fine to me as well, aside from the one grammatical bit I noted.

**Comment by vpodzime** [comment]
Fixed, thanks!

**Pull request merged**